Automattic/pocket-casts-android
# Transcripts - Talkback

## Description
This fixes talkback issues for the transcript screen.

## Testing Instructions
1. Enable talkback
2. Play an episode with transcript
3. Open transcript view
4. Tap on different elements on the screen
5. ✅ Notice that screen elements can be selected and talkback works


** As a side-effect, [this accessibility issue](https://github.com/Automattic/pocket-casts-android/issues/2484) is also resolved, where selecting artwork on the player screen would highlight entire screen and talkback would say unlabeled. 

## Screenshots or Screencast 

https://github.com/user-attachments/assets/9b8d2c72-2f31-48ed-83cb-d86a2912b40b

## Checklist
 
#### I have tested any UI changes...
